### ISSUE TITLE
Flatten arrays

### ISSUE THREAD

**Issue by adius** [adius/shaven#9]
```js
['div',
    [
        ['p', 'foo'],
        ['p', 'bar']
    ]
]
```

should yield the same html as

```js
['div',
    ['p', 'foo'],
    ['p', 'bar']
]
```

**Comment by adius** [comment]
Use es6 destructuring as a workaround:

```js
let content = [
  ['p', 'foo'],
  ['p', 'bar']
]

['div', ...content]
```

**Comment by adius** [comment]
Fixed in 43897db42fe2e2fec42592c21728545e703af78d
